src/client/options/mod.rs

```diff
@@ -500,8 +501,11 @@ pub struct ClientOptions {
     /// supported and is considered undefined behaviour. To run any command with a different API
     /// version or without declaring one, create a separate client that declares the
     /// appropriate API version.
-    #[builder(default, setter(skip))]
-    pub(crate) server_api: Option<ServerApi>,
+    ///
+    /// For more information, see the [Versioned API](
+    /// https://docs.mongodb.com/v5.0/reference/versioned-api/) manual page.
```

Comment:
Can we add this link to the `ServerApi` struct declaration too?

Reply:
Done.